[WIP]Move group interface

anasarrak opened the pull request:
Just ported Plan and execute functions, the only needed ones for the Demo

anasarrak committed:
Starting point with move_group_interface

anasarrak committed:
Remove TODO and add the execution_event_topic

anasarrak committed:
Merge branch 'master' into move_group_interface

anasarrak committed:
Enable getParam and setParam

anasarrak committed:
Change getInterfaceDescription service callback

anasarrak committed:
Update getPlannerParams service callback

anasarrak committed:
Fix getInterfaceDescription service

anasarrak committed:
Fix getPlannerParams services

anasarrak committed:
Update setPlannerParams

anasarrak committed:
Remove return from setPlannerParams

anasarrak committed:
Port execute

anasarrak committed:
Enable all execute part

LanderU commented:
OS X:

```bash
/tmp/ros2_ws/src/moveit2/moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp:161:5: error: no matching member function for call to 'waitForAction'
    waitForAction(move_action_client_, move_group::MOVE_ACTION, timeout_for_servers, allotted_time);
```

anasarrak committed:
Merge branch 'master' into move_group_interface

anasarrak committed:
Fix time on waitForAction

anasarrak commented:
Linux passing, OSX same error as https://github.com/AcutronicRobotics/moveit2/pull/60#issuecomment-488581110

anasarrak committed:
Keep porting move_group_interface

The most essential ones: pick, place, getCurrentState, constructGoal.

Also changed the node_handle_ name to node_

anasarrak committed:
Merge branch 'master' into move_group_interface

anasarrak committed:
Port Move and update return Error result

anasarrak committed:
Merge branch 'move_group_interface' of https://github.com/AcutronicRobotics/moveit2 into move_group_interface

anasarrak committed:
Port stop()

ahcorde committed:
resource retriever from sources

Update osx-compile-moveit2.sh

Update osx-install-ros2-binary.sh

Update moveit2.repos

Update package.xml

Update osx-compile-moveit2.sh

LanderU commented on osx-compile-moveit2.sh:
```diff
@@ -8,7 +8,7 @@ cd /tmp
 mkdir -p /tmp/ros2_ws/src
 cp -r $dir /tmp/ros2_ws/src/moveit2
-cd /tmp/ros2_ws && wget https://raw.githubusercontent.com/AcutronicRobotics/moveit2/master/moveit2.repos
+cd /tmp/ros2_ws && wget https://raw.githubusercontent.com/AcutronicRobotics/moveit2/constraint_sampler_manager_loader_osx/moveit2.repos
```
Make sure to change this before merge it.

ahcorde replied:
Sure

ahcorde commented:
CI is passing in MAC and Linux
